openstack/compute/v2/flavors/results.go

```diff
@@ -28,6 +30,12 @@ type DeleteResult struct {
 	gophercloud.ErrResult
 }
 
+// AccessResult is the result from an Access operation. Call its Extract method
+// to interpret it as a FlavorAccess.
+type AccessResult struct {
+	gophercloud.Result
+}
```

Comment:
This is leftover from me doing a test with `Extract` on `AccessResult`. I'll remove this at the same time any other changes are requested.